website/docs/d/iam_trusted_profile_identities.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile_identities"
+description: |-
+  Get information about iam_trusted_profile_identities
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile_identities
+
+Provides a read-only data source for iam_trusted_profile_identities. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_iam_trusted_profile_identities" "iam_trusted_profile_identities" {
+	profile_id = "profile_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `profile_id` - (Required, Forces new resource, String) ID of the trusted profile.
```

Comment:
For datasources "Forces new resource" is not valid

Reply:
changes have been accomodated.